imu_topic: "/imu/data"
event_left_topic: "/prophesee/left/events"
event_right_topic: "/prophesee/right/events"
camera_left_topic: "/camera/left/image_mono"
camera_right_topic: "/camera/right/image_mono"
kinect_color_topic: "/kinect/color_image"
kinect_depth_topic: "/kinect/depth_image"
lidar_topic: "/ouster/point_cloud"
ground_truth_topic: "/gt/pose" 
event_left_depth_topic: "/prophesee/left/depth_image_undistort"
event_right_depth_topic: "/prophesee/right/depth_image_undistort"
camera_left_depth_topic: "/camera/left/depth_image_undistort"
camera_right_depth_topic: "/camera/right/depth_image_undistort"
camera_left_undistort_topic: "/camera/left/image_mono_undistort"
camera_right_undistort_topic: "/camera/right/image_mono_undistort"
